app/assets/javascripts/discourse/tests/helpers/rich-editor-helper.gjs

```diff
@@ -2,12 +2,7 @@ import { tracked } from "@glimmer/tracking";
 import { click, render, settled, waitFor } from "@ember/test-helpers";
 import DEditor from "discourse/components/d-editor";
 
-export async function testMarkdown(
-  assert,
-  markdown,
-  expectedHtml,
-  expectedMarkdown
-) {
+export async function setupRichEditor(assert, markdown, multiToggle = false) {
```

Comment:
@renato I refactored this a bit so we can set up the rich editor component itself easily for tests like `details` needs where we want to test the interactivity of the stuff inside the editor